Engine/entry1.c

```diff
@@ -1322,6 +1345,12 @@ OENTRY opcodlst_1[] = {
   { "compileorc",  S(COMPILE), 0,  "i", "S",  (SUBR) compile_orc_i, NULL, NULL },
   { "compilecsd",  S(COMPILE), 0,  "i", "S",  (SUBR) compile_csd_i, NULL, NULL },
   { "compilestr",  S(COMPILE), 0,  "i", "S",  (SUBR) compile_str_i, NULL, NULL },
+  { "createinstr",  S(CINSTR), 0,  ":InstrDef;", "S",  (SUBR) compile_instr, NULL, NULL },
+  { "deleteinstr",  S(DELETEIN), 0, "", ":InstrDef;",  NULL, (SUBR) delete_instr, NULL },
+  { "remove",  S(DELETEIN), 0, "", ":InstrDef;",  (SUBR) delete_instr, NULL },
+  { "lambda",  S(CARINSTR), 0,  "", "Sm",  (SUBR) compile_and_run_instr, NULL, NULL },
```

Comment:
I think we should remove these lambda opcodes here or at least rename it as it's not really a lambda. I don't think it adds much compared to using schedule with createinstr. We should probably reserve lambda for anonymous opcodes and it should be a datatype rather than an opcode.